developers/transforms/distributions/index.qmd

```diff
@@ -0,0 +1,314 @@
+---
+title: "Distributions and the Jacobian"
+engine: julia
+---
+
+```{julia}
+#| echo: false
+#| output: false
+using Pkg;
+Pkg.instantiate();
+```
+
+This series of articles will seek to motivate the [Bijectors.jl package](https://github.com/TuringLang/Bijectors.jl/), which provides the tools for transforming distributions in the Turing.jl probabilistic programming language.
+
+It assumes:
+
+- some basic knowledge of probability distributions (the notions of sampling from them and calculating the probability density function for a given distribution); and
+- some calculus (the chain and product rules for differentiation, and changes of variables in integrals).
+
+```{julia}
+import Random
+Random.seed!(468);
+
+using Distributions: Normal, LogNormal, logpdf, Distributions
+using Plots: histogram
+```
+
+## Sampling from a distribution
+
+To sample from a distribution (as defined in [Distributions.jl](https://juliastats.org/Distributions.jl/)), we can use the `rand` function.
+Let's sample from a normal distribution and then plot a histogram of the samples.
+
+```{julia}
+samples = rand(Normal(), 5000)
+histogram(samples, bins=50)
+```
+
+(Calling `Normal()` without any arguments, as we do here, gives us a normal distribution with mean 0 and standard deviation 1.)
+If you want to know the probability of observing any of the samples, you can use `logpdf`:
+
+```{julia}
+println("sample: $(samples[1])")
+println("logpdf: $(logpdf(Normal(), samples[1]))")
+```
+
+The probability density function for the normal distribution with mean 0 and standard deviation 1 is
+
+$$p(x) = \frac{1}{\sqrt{2\pi}} \exp{\left(-\frac{x^2}{2}\right)},$$
+
+so we could also have calculated this manually using:
+
+```{julia}
+log(1 / sqrt(2π) * exp(-samples[1]^2 / 2))
```

Comment:
I know you've just got this here as an example, but would you be up for adding a footnote that warns people against computing the logpdf in this way? i.e. they really ought to simplify the expression first to avoid ever exponentiating -x^2/2

Reply:
I like this!